src/libPMacc/include/particles/ParticlesBase.kernel

```diff
@@ -53,196 +53,150 @@ DINLINE bool getPreviousFrameAndRemoveLastFrame(typename T_ParticleBox::FrameTyp
  * This kernel can only run with a double checker board
  */
 template<class FRAME, class Mapping>
-__global__ void kernelShiftParticles(ParticlesBox<FRAME, Mapping::Dim> pb, Mapping mapper)
+__global__ void kernelShiftParticles( ParticlesBox<FRAME, Mapping::Dim> pb, Mapping mapper )
 {
 
-    using namespace particles::operations;
-
-    /* Exchanges in 2D=8 and in 3D=26
-     */
     enum
     {
         TileSize = math::CT::volume<typename Mapping::SuperCellSize>::type::value,
         Dim = Mapping::Dim,
+        /* Exchanges in 2D=9 and in 3D=27 */
         Exchanges = traits::NumberOfExchanges<Dim>::value
     };
 
 
-    //\todo: testen ob es schneller ist, erst zu flushen wennn Source voll ist
-    __shared__ FRAME * destFrames[Exchanges];
+    /* define memory for two times Exchanges
+     * index [0,Exchanges-1] is called first frame
```

Comment:
is called [for the?] first frame

but even with the addition, I can not understand this comment. can you pls try to rephrase it?